gprofiler/main.py

```diff
@@ -431,9 +431,10 @@ def parse_cmd_args() -> configargparse.Namespace:
         "--nodejs-mode",
         dest="nodejs_mode",
         default="disabled",
-        choices=["perf", "disabled", "none"],
-        help="Select the NodeJS profiling mode: perf (run 'perf inject --jit' on perf results, to augment them"
-        " with jitdump files of NodeJS processes, if present) or disabled (no runtime-specific profilers for NodeJS)",
+        choices=["attachable", "perf", "disabled", "none"],
```

Comment:
Let's call it `attach` not `attachable`, and maybe `attach-maps` (and later we'll add `attach-jit` for jitdumps)